Plant-for-the-Planet-org/planet-webapp
# Treemapper Import Web

### Things to test -
- Treemapper Import - User Dashboard -> Treemapper -> Import
- Plant Date Selection - no future date
- Project Selection if you are a TPO
- Upload geojson/json editor/draw on map
- Add species name and count
- Continue
- Download sample tree template
- upload filled template
- check for validation errors
- edit values before upload
- delete some sample trees
- continue
- try skipping sample trees
- check for successful upload message
- go to my plant locations
